drivers/sx126x/sx126x.c

```diff
@@ -204,52 +223,61 @@ int sx126x_init(sx126x_t *dev)
 
 uint32_t sx126x_get_channel(const sx126x_t *dev)
 {
+    DEBUG("[sx126x.c] sx126x_get_radio_status \n");
     return dev->channel;
 }
 
 void sx126x_set_channel(sx126x_t *dev, uint32_t freq)
 {
+    DEBUG("[sx126x.c] sx126x_get_channel \n");
```

Comment:
Now that you are adding so many debug messages, for completeness I would try to print the values on the `set`s when possible.

Reply:
That would make sense :)